docs/concepts/models/overview.md

```diff
@@ -141,7 +141,7 @@ For models that are incremental, the following parameters can be specified in th
 - Engines that support partitioning, such as Spark and BigQuery, use the time column as the model's partition key. Multi-column partitions or modifications to columns can be specified with the [`partitioned_by` property](#partitioned_by).
 
 ### lookback
-- Lookback is used for [incremental](model_kinds.md#incremental_by_time_range) models to capture late arriving data. This must be a positive integer and refers to the number of units that late arriving data is expected.
+- Lookback is used with [incremental by time range](model_kinds.md#incremental_by_time_range) models to capture late-arriving data and/or execute calculations requiring historical data ("rolling window" calculations). It must be a positive integer and specifies the number of interval time units prior to the current interval the model should include. For example, a model with cron `@daily` and `lookback` of 7 would include the previous 7 days each time it ran, while a model with cron `@weekly` and `lookback` of 7 would include the previous 7 weeks each time it ran.
```

Comment:
Not sure about this part:

> and/or execute calculations requiring historical data ("rolling window" calculations). It

I think we learnt today that `lookback` is not a good fit for this specific purpose. @tobymao wdyt?